sql/src/main/java/io/crate/operation/collect/JobCollectContext.java

```diff
@@ -200,6 +200,14 @@ public SharedShardContexts sharedShardContexts() {
         return sharedShardContexts;
     }
 
+    public CollectPhase collectPhase() {
+        return collectPhase;
+    }
+
+    public void collectPhase(CollectPhase collectPhase) {
```

Comment:
hmmm maybe you're right, but then I cannot do this:
https://github.com/crate/crate/pull/4462/files#diff-83c6d939db6fc42a4fe9c29ad0fa3512R96